src/leo-server-app/src/models/schedule.ts

```diff
@@ -0,0 +1,29 @@
+import mongoose from "mongoose";
+import { ScheduleType } from "../types/schedule";
+
+const Schema = mongoose.Schema;
+
+const scheduleSchema = new Schema(
+  {
+    commands: Schema.Types.Mixed,
```

Comment:
why not define this as an array of commands? that way we can iterate through them in the future

Reply:
oops should have been, fixed